apps/remix-ide/src/lib/helper.js

```diff
@@ -68,7 +68,16 @@ module.exports = {
     const hexValue = hash.slice(2, hash.length)
     return this.is0XPrefixed(hash) && /^[0-9a-fA-F]{64}$/.test(hexValue)
   },
-  find: find
+  find: find,
+  getPathIcon (path) {
+    return path.endsWith('.txt')
```

Comment:
that would be perhaps more readable with a `switch` `case`

Reply:
A switch statement will not work so well for `path.endsWith()` because the argument for switch will need to match a case, which we don't have in this case.